### ISSUE TITLE
Cond/Map and Parse use

### ISSUE THREAD

**Issue by r4v3n6101** [rust-bakery/nom-derive#2]
I can't use Cond, Map and Parse at the same time. Such code won't be compiled:
```rust
#[nom(
    Cond = "a == 0"
    Parse = "take_till(|b| b == b'0')",
    Map = "bytes_to_cstring"
)]
cstring: Option<CString>
```

**Comment by chifflier** [comment]
Thanks for the report!

Note: a comma is missing after `Cond` (`Cond = "a == 0",`)

I get the following error:
```
message: A condition was given on field cstring, which is not an option type.
Hint: use Option<...>
```

It seems the parser does not get the type correctly. I'll investigate

**Comment by chifflier** [comment]
Fixed in f8cdf023f0c35c8fe202c7d7ebcf88db64f06e70

The tested parser (added in unit tests) is:
```rust
#[derive(Debug, Nom)]
pub struct MultipleAttributes1 {
    pub a: u32,
    #[nom(
        Cond = "a == 0",
        Parse = "take_till(|b| b == 0)",
        Map = "bytes_to_cstring"
    )]
    cstring: Option<CString>,
}
```

(note: `take_till` leaves a trailing `\0` unparsed, I don't know if that was intended here)
